docs/plugin-development.rst

```diff
@@ -389,6 +389,18 @@ You can use all of the testing utilities provided by the ``base.TestCase`` class
 from core. You will also get coverage results for your plugin aggregated with
 the main Girder coverage results if coverage is enabled.
 
+Plugins can also use the external data interface provided by Girder as described
+in :ref:`use_external_data`.  For plugins, the data key files should be placed
+inside a directory called ``plugin_tests/data/``.  When referencing the
+files, they must be prefixed by your plugin name as follows
+
+.. code-block:: cmake
+
+    add_python_test(my_test EXTERNAL_DATA plugins/cats/test_file.txt)
```

Comment:
That is possible, but it really begins to negate the advantage of using `ExternalData` in the first place.  If we are going to go through the file list and search for key files on our own, we are reimplementing a sizable portion of the original module.  It would also ruin much of the extensibility that I tried to bake in.  That is, you can still use functions like `girder_add_test` as you would use `ExternalData_add_test`.  The more magic we try to bake in the more we will break from the standard module and reimplement for ourselves.